Documentation/compatibility/winforms-accessibility-changes-472.md

```diff
@@ -23,8 +23,9 @@ __How to opt in or out of these changes__
   
 In order for the application to benefit from these changes, it must run on the .NET Framework 4.7.2 or later. The application can benefit from these changes in either of the following ways:
 - It is recompiled to target the .NET Framework 4.7.2. These accessibility changes are enabled by default on Windows Forms applications that target the .NET Framework 4.7.2 or later.
-- It opts out of the legacy accessibility behaviors by adding the following [AppContext Switch](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the ```<runtime>``` section of the app config file and setting it to false, as the following example shows.
-```

```

Comment:
Eek, I thought I caught that one and got rid of it. :) It took me a moment to figure out what you were referring to as inline (I'd thought you meant the code snippet) but I did eventually. I could have sworn I pushed a change to that, but it could well have been in the other doc.  Thanks!!